### DISCUSSION TITLE
band scale - can't hide ticks

### DISCUSSION THREAD

**Discussion by ebeling** [observablehq/plot#1048] [Q&A] [answered]
I'm using plot in an Angular (Typescript) - project. I like to hide ticks in a plot with band scale on x-axis. The label must be visible. 
``` ...
const data = this.initTestData();
const svg = Plot.plot({
        height: this.height,
        width: this.width,       
        grid: false,
        marks: [
          Plot.barY(data, {
                      x: (d: ISpiritData) => {
                        return this.getXValue(d);
                      },
                      y: (d: ISpiritData) => {
                        return this.getYValue(d);
                      },
                      rx: 1,
                      fill: (d: ISpiritData) => {
                               return this.getColor(d);
                            }
          }),
      x: {
          type: 'band',
          ticks: 0,
          tickSize: 0,         
          line: false,
           label: 'my label',
            },
        y: {
            axis: null
           }
    });
....
```
The ticks on x-scale are visible. What is wrong?
using Angular 14.2.1, d3 7.6.1, @observablehq/plot 0.6.0

**Comment by Fil** [comment]
Can you share a reproducible case? The following is working for me: https://observablehq.com/@recifs/band-scale-ticksize--support

**Comment by ebeling** [comment]
Thank you for your fast reply. If I use "Plot.dot" in my tests it works for me, but if I use "Plot.barY" it doesn't. I don't want to see ticks. Is it enough if I show you, how I init the test data or do you need more?

**Comment by Fil** [comment]
Even with barY it works for me, as you can see on https://observablehq.com/@recifs/band-scale-ticksize--support

Please share a fully-working reproducible example (in an observable notebook or codepen); otherwise I have to do guesswork from incomplete information, and it takes a lot of effort. In particular, please make sure that the syntax is correct—the code above seems to have the definition for x located inside the marks array.

**Comment by ebeling** [comment]
Here is my example: https://stackblitz.com/edit/observablehq-plot-test-ezsagj?file=src/app/app.component.ts
How can I hide the dates on x-axis?

**Comment by Fil** [comment, marked as answer]
you could use tickFormat: () => ""

**Reply by ebeling** [reply]
Yes, this works! Thank you very much!

**Comment by dgwyer** [comment]
Or, if you wanted to display the day in narrow format, this works well.

```
tickFormat: d => d.toLocaleString("en", {weekday: "narrow"}),
```
